src/Widgets/SaveDialog.vala

```diff
@@ -85,6 +85,10 @@ namespace Screenshot.Widgets {
             /// TRANSLATORS: %s represents a timestamp here
             var file_name = _("Screenshot from %s").printf (date_time);
 
+            if (this.scale_factor > 1) {
+                file_name += " @%d".printf (this.scale_factor);
```

Comment:
Do not use `%d`, use `%i`.

Reply:
ok, but what is the different?

Reply:
Is scale factor guaranteed to be an integer? I know GNOME has implemented non-integer-scaling, so we might not want to assume an integer to be the most compatible/future-proof.

Reply:
I guess according to Valadoc it's an int, so never mind.

Reply:
current property is an int:
https://valadoc.org/gtk+-3.0/Gtk.Widget.scale_factor.html